Modules/_testcapimodule.c

```diff
@@ -4188,6 +4188,58 @@ test_pymem_alloc0(PyObject *self, PyObject *Py_UNUSED(ignored))
     Py_RETURN_NONE;
 }
 
+static PyObject *
+test_pymem_new(PyObject *self, PyObject *Py_UNUSED(ignored))
+{
+#define CHECK_PTR(ptr) \
```

Comment:
Unused macro?

Reply:
Looks like leftovers from an early revision. I'm sure Victor will remove it before merging.

Reply:
Oops, I forgot about it. That's why reviews are great :-) I removed this unused macro. I used if+goto instead of the macro.